internal/etl/transform/extend_values.go

```diff
@@ -2,24 +2,52 @@ package transform
 
 import (
 	"encoding/csv"
+	"fmt"
 	"os"
+	"slices"
 	"strings"
 )
 
-func generateFieldValuesSubstitutions(fieldValuesByName map[string]any, substitutionFile string) ([]map[string]any, error) {
-	substitutions, err := readSubstitutionsFile(substitutionFile)
+// Return slice of fieldValuesByName
+func generateAllFieldValues(fieldValuesByName map[string]any, substitutionsFile, synonymsFile string) ([]map[string]any, error) {
+	substitutions, err := readSubstitutionsFile(substitutionsFile)
 	if err != nil {
 		return nil, err
 	}
-	var fieldValuesByNameExtensions = make(map[string][]string)
+	synonyms, err := readSubstitutionsFile(synonymsFile)
+	if err != nil {
+		return nil, err
+	}
+
+	var fieldValuesByNameWithAllValues = make(map[string][]string)
 	for key, value := range fieldValuesByName {
-		valueSubstitutions, err := applySubstitutions(value.(string), substitutions)
+		valueLower := strings.ToLower(value.(string))
+
+		// Get all substitutions
+		substitutedValues, err := extentValues([]string{valueLower}, substitutions)
```

Comment:
extent vs extend (filename is extend). Moet extend zijn volgens mij (snap de extent wel gezien de geo werkzaamheden 😉 )

Reply:
:) deze zat op meer plekken. DONE